src/main/java/edu/cornell/kfs/concur/ConcurConstants.java

```diff
@@ -53,4 +53,25 @@
         public static final String CREDIT = "CR";
         public static final String DEBIT = "DR";
     }
+    
+    public class StandardAccountingExtractPdpConstants {
+        public static final String PAYMENT_CODE_CASH = "CASH";
+        public static final String COMBINED_GROUP_INDICATOR = "Y";
+        public static final String BANK_CODE = "DISB";
+        public static final String EMPLOYEE_STATUS_CODE = "EMPLOYEE";
+        public static final String EMPLOYEE_PAYEE_STATUS_TYPE_CODE = "E";
+        public static final String NON_EMPLOYEE_PAYEE_STATUS_TYPE_CODE = "Y"; 
+        public static final String FEED_FILE_ENTRY_HEADER_VERSION = "1.0";
+    }
+    
+    public static final String DEFAULT_ORIGINATION_CODE_PARAMETER_NAME = "CONCUR_AP_PDP_ORIGINATION_CODE";
```

Comment:
On the same note, I think usually the constant name and parameter name are the same.